src/ci/docker/host-aarch64/aarch64-gnu-debug/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM ubuntu:22.04
+FROM ubuntu:25.04
 
 ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install -y --no-install-recommends \
-  g++ \
+  g++-15 \
```

Comment:
Question: ... and for that matter, is this a higher g++ baseline? (I'm not super sure.)